ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouter.java

```diff
@@ -46,6 +47,8 @@
 class NonBlockingRouter implements Router {
   private final NetworkClientFactory networkClientFactory;
   private final ArrayList<OperationController> ocList;
+  private final BackgroundDeleter backgroundDeleter;
+  private final int ocCount;
```

Comment:
Felt that it minimizes code bloat since this field of the `routerConfig` is used at a few places.